lib/injectCustomJavaScript.js

```diff
@@ -1,19 +1,19 @@
-import { registerForPushNotificationsAsync } from './pushNotifications';
-
-const handleLoggedInPage = async (webview) => {
-  const token = await registerForPushNotificationsAsync();
-
-  webview.injectJavaScript(`window.TimeOverflowRegisterExpoDeviceToken(\'${token}\');`);
+const handleLoggedInPage = (webview, token) => {
+  webview?.injectJavaScript(
+    `window.TimeOverflowRegisterExpoDeviceToken(\'${token}\');`
+  );
 };
 
 const handleLoginPage = (webview) => {
-  webview.injectJavaScript("$('#user_remember_me').prop('checked', true);")
+  webview?.injectJavaScript("$('#user_remember_me').prop('checked', true);");
```

Comment:
@sauloperez maybe we can write this small js expression without jQuery, using vanilla js? So if we eventually remove jQuery from main app, this will continue to work.

Reply:
yeah, I think we can clearly do that in Vanilla JS. I'll look into it.

Reply:
@sauloperez I think this change should be enough:

```suggestion
  webview?.injectJavaScript("document.querySelector('#user_remember_me').checked = true");
```

Reply:
yep, I think so. I'll try it out as soon as I can get back to this.